azerothcore/azerothcore-wotlk
# fix(DB/loot): Adjust Nightbane Dark Runner loot

A reference table (id: 24042) containing level 40 items was included
in Nightbane Dark Runner's (id: 205) (level 28-29) loot table. This
seems wrong and it *is* wrong according to wowhead classic, so I removed
the reference.



## Changes Proposed:
- Removed pointer to reference table 24042 from the Nightbane Dark Runner loot table

## Issues Addressed:

- Closes #5965
- Closes https://github.com/chromiecraft/chromiecraft/issues/675

## SOURCE:

https://classic.wowhead.com/npc=205/nightbane-dark-runner

## Tests Performed:

- Made sure the sql applies cleanly and double checked that the correct ref table has been removed from the correct mob

## How to Test the Changes:


1. Check the new loot table; make sure creature id 205 loot table doesn't include reference table 24042


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
